### PR TITLE
XCast Json and md files updated as per code

### PR DESCRIPTION
**Pull request by pjayar213**
Documentation enhancement for XCast plugin
Jira ticket link: https://ccp.sys.comcast.net/browse/RDK-35237

### CODE REVIEW

File: XCast/XCast.json [hunk 1]
```diff
@@ -161,7 +161,7 @@
             }
         },
         "registerApplications": {
-            "summary": "Registers an application",
+            "summary": "Registers an application.\n  \n### Events \n\n No Events",
```
Comment (apatel859) [inline XCast/XCast.json:164]: This API allows to whitelist the apps which support dial service. If app want to dynamically update the app list same API should be called with updated list.

File: XCast/XCast.json [hunk 2]
```diff
@@ -197,7 +197,7 @@
             }
         },
         "setFriendlyName":{
-            "summary": "Sets a name for the implementation",
+            "summary": "Sets a name for the implementation.\n  \n### Events \n\n No Events",
```
Comment (apatel859) [inline XCast/XCast.json:200]: This API allows application to override the default friendly name value with the friendly name  passed as an argument.

File: XCast/XCast.json [hunk 3]
```diff
@@ -94,7 +94,7 @@
             }
         },
         "getFriendlyName":{
-            "summary": "Gets the human readable name for the implementation",
+            "summary": "Gets the human readable name for the implementation.\n  \n### Events \n\n No Events",
```
Comment (apatel859) [inline XCast/XCast.json:97]: Returns the friendly name set by setFriendlyName API.

File: XCast/XCast.json [hunk 4]
```diff
@@ -214,7 +214,7 @@
             }
         },
         "setStandbyBehavior":{
-            "summary": "Sets the expected xcast behavior in standby mode",
+            "summary": "Sets the expected xcast behavior in standby mode.\n  \n### Events \n\n No Events",
```
Comment (apatel859) [inline XCast/XCast.json:217]: This API allows application to override controls the xcast behavior in standby mode. Default behavior in STANDBY mode is disable so client device can not discover the server. When STANDBY behavior is active client device could discover the server.

File: XCast/XCast.json (hunk 1 above)
Comment (pjayar213) [reply XCast/XCast.json]: Updated changes as per your review comments

File: XCast/XCast.json (hunk 2 above)
Comment (pjayar213) [reply XCast/XCast.json]: Updated changes as per your review comments

File: XCast/XCast.json (hunk 3 above)
Comment (pjayar213) [reply XCast/XCast.json]: Updated changes as per your review comments

File: XCast/XCast.json (hunk 4 above)
Comment (pjayar213) [reply XCast/XCast.json]: Updated changes as per your review comments

**Review by apatel859 — APPROVED** [review]

**Review by anand-ky — APPROVED** [review]

**Pull request merged**